installer/manifests/common/templates/_images.tpl

```diff
@@ -0,0 +1,39 @@
+{{/*Return the proper serivce image name*/}}
+{{- define "service.image" -}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
+{{- $registryName := "" -}}
+{{- $repositoryName := .imageRoot.repository -}}
+{{- $tag := include "service.tag" (dict "imageRoot" .imageRoot "global" .global "defaultTag" .defaultTag) -}}
+{{- if .global }}
+    {{- if .global.registry }}
+     {{- $registryName = .global.registry -}}
+    {{- end -}}
+{{- end -}}
+{{- if .imageRoot.registry -}}
+  {{- $registryName = .imageRoot.registry -}}
+{{- end -}}
```

Comment:
In bitnami's common chart, the global settings have precedence over the imageRoot object, why did you do it the other way around here?

Reply:
I think bc we want it the other way around? We always take the global one but if there is a registry set at the imageRoot.registry level then take this instead.
What do you think?

Reply:
Makes sense :)